src/renderer/Light.ts

```diff
@@ -40,7 +40,17 @@ export class Light implements Bakeable {
     constructor(params: LightParams) {
         this.lightPosition = Mapper.coordToVector(params.position);
         this.lightColor = params.color.asVec();
-        this.lightIntensity = params.intensity;
+        this.lightStrength = params.strength;
+    }
+
+    /**
+     * Instantiate a new Light object.
+     *
+     * @param {LightParams} params The named parameters for the light source.
+     * @returns {Light} A new light.
+     */
+    public static create(params: LightParams): Light {
```

Comment:
Do you want to make the constructor `private` if we're using this static method to initialize - or are we keeping both ways of instantiating `Light`s?